Header header
uint32[] mode                         # mapping, repeating
Features[] map_features
Features[] live_features
float32[] map_distances
float32[] map_transitions
float32[] map_similarity
int32[] time_transitions
uint32[] map_indices                   # map_id, number of maps, closest img idx